.github/workflows/gradle.yml

```diff
@@ -9,9 +9,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - name: Set up JDK 1.8
+    - name: Set up JDK
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
```

Comment:
I was thinking whether to target 1.8 or 11.

And I'm not sure. 8 is still very popular.

Reply:
Done ;)